docs/canvas/canvas-workpad.asciidoc

```diff
@@ -69,25 +69,68 @@ Each of the sample data sets comes with a Canvas workpad that you can use for yo
 
 . Add a {kibana-ref}/add-sample-data.html[sample data set].
 
-. On the *Add Data to Kibana* page, click the *View data* dropdown list, then select *Canvas*.
-+
-Need some more workpad inspiration? Check out the link:https://www.elastic.co/blog/[Elastic Blog].
+. On the *Add Data* page, click the *View data* dropdown list, then select *Canvas*.
 
 [float]
 [[apply-workpad-styles]]
 == Apply a set of styles to the entire workpad
```

Comment:
Should this and the rest of the headings below this be h3 to match the headings above it?